### DISCUSSION TITLE
Terminal Chest

### DISCUSSION THREAD

**Discussion by KohlrabitoDev** [Kwantux/networks#54] [Ideas]
Maybe something kinda like the AE2 terminals, you could make a chest that basically does the same thing as /net view but allows you to see every single item in your network but with add/remove access. I don't know how it would work exactly but maybe you could also have it to stack everything together & have it in some organized list.

**Comment by Kwantux** [comment]
Hey, sorry for the late response, it seems that my notification settings were incorrectly configured.

This is indeed actually planned for the future.
For now a few other updates have more priority, but this will come for sure.
